### PR TITLE
ktor: split to netty and jetty tests, configure mysql (disable ssl)

### PR DESCRIPTION
**Pull request by cy6erGn0m**
- run ktor tests with both jetty and netty backends
  * build separate jars
  * use ktor development launcher
- disable SSL
- upgrade kotlin (1.1.2-5) and ktor (0.3.3)

### CODE REVIEW

File: frameworks/Kotlin/ktor/benchmark_config.json [hunk 1]
```diff
@@ -2,15 +2,15 @@
   "framework": "ktor",
   "tests": [
     {
-      "default": {
+      "netty": {
```
Comment (nbrady-techempower) [inline frameworks/Kotlin/ktor/benchmark_config.json]: We still need a test marked as `default`.

Comment (cy6erGn0m) [inline frameworks/Kotlin/ktor/benchmark_config.json]: so if I have one `"default"` and one `"jetty"` then what test will be started if I specify no test name? (like `tfb -m benchmark`) Both or only the default one?

Comment (nbrady-techempower) [inline frameworks/Kotlin/ktor/benchmark_config.json]: The `default` test is the one that gets run.:

`tfb --test ktor` - runs the default test
`tfb --test ktor-netty` - runs the netty test in this case
`tfb --test-dir Kotlin/ktor` - would run all the tests in your benchmark config

File: frameworks/Kotlin/ktor/benchmark_config.json [hunk 2]
```diff
@@ -21,7 +21,30 @@
...
+        "classification": "Micro",
+        "database": "MySQL",
+        "framework": "ktor",
+        "language": "Kotlin",
+        "orm": "Raw",
+        "platform": "Jetty",
+        "webserver": "None",
+        "os": "Linux",
+        "database_os": "Linux",
+        "display_name": "ktor netty",
```
Comment (nbrady-techempower) [inline frameworks/Kotlin/ktor/benchmark_config.json]: You also probably wanted `"display_name": "ktor jetty",` here

**Comment by cy6erGn0m** [comment]
@nbrady-techempower anything else to be done?

**Pull request merged**